batchflow/pipeline.py

```diff
@@ -764,16 +764,16 @@ def _exec_all_actions(self, batch, actions=None):
         actions = actions or self._actions
 
         for action in actions:
+            if self._profile:
+                start_time = time.time()
+                self._profiler.enable()
```

Comment:
Perhaps it might be convenient to split arg evaluation and action execution for profiling puproses